bot/code_review_bot/__init__.py

```diff
@@ -109,6 +109,19 @@ def as_phabricator_lint(self):
         '''
         raise NotImplementedError
 
+    def as_phabricator_unitresult(self):
+        '''
+        In case this issue is a build failure get the UnitResult
```

Comment:
The comment seems a bit too specific for the base implementation. It should say something lile: `Build a Phabricator UnitResult to publish through Harbormaster API`